docs/source/guide/build_pipelines/pipeline_basics.rst

```diff
@@ -0,0 +1,259 @@
+.. _pipeline_basics:
+
+Basics
+======
+
+Data science workflows revolve around building and refining pipelines, i.e., a series of processes that transform data into useful information/product
+(read more about pipelines :ref:`here <concepts>`).
+
+Traditionally, this is often manual and time-consuming work as data scientists (or production engineers) need to transform messy development code
+into deployable scripts for the target system (e.g., Airflow).
+
+Having the complete development process stored in artifacts, LineaPy can automate such code transformation, accelerating transition from development to production.
+
+For example, consider a simple pipeline that 1) pre-processes raw data and 2) trains a model with the pre-processed data.
+
+.. image:: pipeline.png
+  :width: 600
+  :align: center
+  :alt: Pipeline Example
+
+Pipeline Creation
+-----------------
+
+Once we have the pre-processed data and the trained model stored as LineaPy artifacts (which can be done during development sessions),
+building a pipeline reduces to “stitching” these artifacts, like so:
+
+.. code:: python
+
+   import lineapy
+
+   # Load artifacts to use in pipeline building
+   preprocessing_art = lineapy.get("iris_preprocessed")
+   modeling_art = lineapy.get("iris_model")
+
+   # Build an Airflow pipeline using artifacts
+   lineapy.to_pipeline(
+      pipeline_name="iris_pipeline",
+      artifacts=[preprocessing_art.name, modeling_art.name],
+      dependencies={modeling_art.name: {preprocessing_art.name}},
+      output_dir="./output/pipeline_basics/",
+      framework="AIRFLOW",
+   )
+
+where ``{modeling_art.name: {preprocessing_art.name}}`` is a way to indicate that
+the trained model depends on the pre-processed data (check :func:`lineapy.to_pipeline`
```

Comment:
trained model 'artifact' maybe? assuming you remove the two lines of lineapy.get you might need to make this clarification.

Reply:
Yes, making the update!